library/src/test/java/org/apache/apex/malhar/lib/state/spillable/SpillableSetMultimapImplTest.java

```diff
@@ -185,24 +205,8 @@ public long simpleMultiKeyTestHelper(SpillableStateStore store,
     SpillableTestUtils.checkValue(store, 0L,
         SliceUtils.concatenate(keyBytes, SpillableSetMultimapImpl.SIZE_KEY_SUFFIX), 11, 0, serdeInt);
 
-    map.removeAll(key);
-
-    map.endWindow();
-    store.endWindow();
-
-    nextWindowId++;
-    store.beginWindow(nextWindowId);
-    map.beginWindow(nextWindowId);
-
-    Assert.assertFalse(map.containsKey(key));
-
-    SpillableTestUtils.checkValue(store, 0L,
-        SliceUtils.concatenate(keyBytes, SpillableSetMultimapImpl.SIZE_KEY_SUFFIX), 0, 0, serdeInt);
-
-
     map.endWindow();
     store.endWindow();
-    store.beforeCheckpoint(nextWindowId);
```

Comment:
Why was this removed? In order to correctly simulate a checkpoint with managed state you need to beforeCheckpoint and then checkpointed. It looks like there may be a pre existing issue on line 228 because the window is incremented before checkpointed is called... My bad.